CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.2.0] - January 19, 2021
+### Changed
+- Bump @logdna/logger which includes an `axios` fix for a SSRF vulerability. This bump
```

Comment:
@darinspivey, can you append "[#219] (https://github.com/logdna/logdna-agent/pull/219)" as well like we did for others?

Reply:
I don't follow.  This is 219.  The bump includes the vulnerability patch as well as the passthrough change.

Reply:
fixed